spec/system/page_objects/pages/category.rb

```diff
@@ -56,6 +56,7 @@ def back_to_category
 
       def save_settings
         find("#save-category").click
+        expect(page).to have_css("#save-category", text: I18n.t("js.category.save"))
```

Comment:
Adding this assertion here to make the test fail since the Ember app crashes after saving.